website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,77 @@
+    - To add an external email address, select the **Notification email** dropdown and choose **Add external email**. After you add the external email, it becomes available for selection in the **Notification email** dropdown list.
 
-1. Click the gear menu in the top right corner and select **Notification Settings**.
+    <Lightbox src="/img/docs/deploy/example-notification-external-email.png" width="50%" title="Example of the Notification email dropdown"/>
 
-2. Select **Edit** to begin editing the **Email Notifications** settings.
-    - **As a user:** Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want to receive notifications for.
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
 
-    - **As an admin:**  Under **Configure notifications for**, use the dropdown to select one or more users you'd like to set notifications for. If you only see your own name, then you might not have admin privileges. <br /><br />
-    Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want them to receive notifications for.
+1. Click **Edit** to configure the email notification settings.
 
-3. Click **Save**.
+    - If you're a **user**, choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications for.
 
-    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/email-notifications.png" width="75%" title="Configuring Email Notifications"/>
+    - If you're an **admin**, use the **Configure notifications for** dropdown to select all the users you want to sent notifications to. Then, choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want them to receive notifications for.
 
-### Slack
+1. When you're done with the settings, click **Save**.
+    
+    To set up alerts on jobs from a different environment, select another **Environment** from the dropdown, **Edit** those job notification settings, and click **Save**.
 
-<Snippet path="slack-notifications-config-steps" />
+    <Lightbox src="/img/docs/deploy/example-email-notification-settings-page.png" width="100%" title="Example of the Email notifications page"/>
+
+### Unsubscribe from email notifications
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. On the **Email notifications** page, click **Unsubscribe from all email notifications**. 
+
+## Slack notifications
+
+You can receive Slack alerts by setting up the Slack integration first, then configuring the dbt Cloud notification settings.
+
+:::note 
+If there have been changes to the user roles and you need to move ownership, please [contact support](mailto:support@getdbt.com) and provide the necessary information for them to make this change for you.
+:::
```

Comment:
Maybe something liek this:

"While any account admin can edit slack notifications, they will be limited to configuring the channels to which their Slack user has access to. If there has been a change in user roles or slack permissions such that you no longer have access to edit a configured slack channel, please contact support"